src/gui/mzroll/isotopeswidget.cpp

```diff
@@ -311,7 +311,7 @@ QString IsotopeWidget::groupIsotopeMatrixExport(PeakGroup* group, bool includeSa
                 isotopes.push_back(isotope);
 			}
 		}
-		std::sort(isotopes.begin(), isotopes.end(), PeakGroup::compC13);
+		//std::sort(isotopes.begin(), isotopes.end(), PeakGroup::compC13);
```

Comment:
are we not sure we won't be using it the future ?

Reply:
Yes, there's no point in sorting by C13 count here. The output should be the same as what you see in the Peak Table. Plus, this sorting is already disabled in further functions as well.